L1TriggerConfig/Utilities/src/L1TCaloParamsViewer.cc

```diff
@@ -218,13 +230,13 @@ void L1TCaloParamsViewer::analyze(const edm::Event& iEvent, const edm::EventSetu
 
     if( !ptr1->tauCalibrationLUT()->empty() ){
         cout<<"  tauCalibrationLUT=      ["<<ptr1->tauCalibrationLUT()->maxSize()<<"] "<<flush;
-        int tauCalibration[ptr1->tauCalibrationLUT()->maxSize()];
-        for(unsigned int i=0; i<ptr1->tauCalibrationLUT()->maxSize(); i++)
+        int tauCalibration[512]; //ptr1->tauCalibrationLUT()->maxSize()];
```

Comment:
seems better then to use the maxSize in the loop (checking that it does not exceed your 512 array length limit. Please consider for next PR